### DISCUSSION TITLE
Publication document en open access (pour Open Knowledge Association)

### DISCUSSION THREAD

**Discussion by AbcSxyZ** [Open-Models/Brique#345] [Questions & Answers]
Questions de @slrleao :

> Pour l’open access on doit obligatoirement passer par une plateforme ?

Je suppose que la question est de savoir si le site de votre organisation suffit. L'intérêt de publier sur une archive ouverte, un dépôt en open access est que votre contenu sera indexé sur les moteurs de recherche en open access (type [CORE](https://core.ac.uk)), notamment parce que les dépôts implémentent des standards type OAI-PMH/dublin core pour la communication des informations, ce qui n'est pas le cas d'un site perso.

Il pourrait peut-être intéressant de ne pas directement publier le pdf sur votre site mais de pointer sur une plateforme.

>  Est-ce qu’il faut publier sur un maximum de ces plateformes ?

La question c'est surtout de savoir comment le document sera accessible, dans une archive indexée par les moteurs de recherche open access, un seul endroit suffit. Peut-être que certaines plateformes intéressantes pour les chercheurs ne communiquent pas avec ces systèmes décentralisés de l'open access, type [researchgate](https://en.wikipedia.org/wiki/ResearchGate), un facebook like pour chercheurs. Je ne sais pas vraiment comment ça marche de ce côté là et s'il devient alors pertinent de poster ça à quelques endroits stratégiques en plus.

Le problème c'est de savoir ce qui favorise la découvrabilité du document.

> Doi est une fondation et la majorité des agences sont Chinoise, je suis un peu perdu

DOI est un élément omniprésent pour donner un id unique aux "objets numériques". C'est porté par une fondation et ça doit poser tout un certain nombre de questions que je ne maîtrise pas forcément, mais c'est une pratique aujourd'hui normalisée pour les articles scientifiques. Vous pouvez passer par votre structure (ici pour la [HES-SO](https://www.hes-so.ch/recherche-innovation/open-science/open-access/publier-en-open-access/comment-publier-en-open-access)) pour obtenir un DOI.

Pour les "agences chinoises", je ne suis pas sûr si tu parles des dépôts/archives ouvertes ou par rapport au DOI, en tout cas Zenodo c'est le CERN, Open Science Framework c'est américain. Je ne connais pas vraiment d'outils chinois.

Par rapport à ArXiv, je ne suis pas sûr de la plateforme idéale, mais en tout c'est plus destiné aux preprint, ce qui ne sera pas forcément indéfiniment le statut de votre contenu, mais ça pose la question de savoir à quelle moment on sort d'un statut "preprint". Il faudrait regarder plus en profondeur la pertinence de la plateforme.

**Comment by slrleao** [comment]
Merci pour ta réponse, je comprends mieux. Ces plateformes s'occupent de faire du "hosting" pour les papers et aider à apparaitre dans les moteurs de recherches. A quel point on peut faire confiance à ces plateformes ? Ca ne me parait pas vraiment décentralisé, peut-être que je me trompe.

Merci pour le lien HES-SO pour le DOI.

Donc il nous faut un DOI plus une publication en open access sur une plateforme type Zenodo par exemple ?

**Reply by AbcSxyZ** [reply]
> Ca ne me parait pas vraiment décentralisé

Les systèmes décentralisés ont des formes de centralisation, pour Internet c'est les "[Autonomous System](https://fr.wikipedia.org/wiki/Autonomous_System)" gérés par des opérateurs ou organisations, les fournisseurs d'accès à Internet.

On est sur un réseau de "bibliothèques", où on peut naviguer dans l'ensemble. Tu peux mettre en place ta bibliothèque, les archives ouvertes peuvent se mettre en place depuis des logiciels open source. Zenodo est une instance du CERN du logiciel open source Invenio.

La décentralisation n'est jamais parfaite :)

Confiance, tout dépend de ce que tu entends et attends, mais rien est éternel et immuable. 

Est-ce que votre site internet est plus pérenne que l'infrastructure de votre établissement ? En l'état, probablement pas. Pour Zenodo, tu as une section sur la [longévité du contenu](https://about.zenodo.org/policies/) :

> Replicas: All data files are stored in CERN Data Centres, primarily Geneva, with replicas in Budapest. Data files are kept in multiple replicas in a distributed file system, which is backed up to tape on a nightly basis.

Non, il ne faut pas un DOI, mais ça facilite de retrouver le travail. Un exemple, dans votre travail il y a le doi 10.1111/1468-2427.12077, tu peux le rentrer là-dedans : https://dx.doi.org/

Zenodo semble en générer automatiquement un, mais il y a zenodo qui apparaît dedans, peut être que ça peut être pas mal de quand même en demander un dans votre structure. La bibliothèque universitaire est aussi un bon endroit pour poser quelques questions et avoir des avis/conseils adaptés à votre structure.

**Reply by AbcSxyZ** [reply]
Je viens même de découvrir qu'il suffit de rajouter un doi à la fin de doi.org : https://doi.org/10.1111/1468-2427.12077

**Reply by slrleao** [reply]
Merci pour les informations et les précisions !

Donc je récapitule:

1. Demander un DOI à la HES-SO
2. Publier le travail sur Zenodo (p.ex)
3. Mettre le document accessible sur notre site web (arrive prochainement)

Concernant notre présentation, je travail sur un concept différent d'une présentation classique, je voulais pas trop de donner d'infos là dessus, pour garder la surprise mais ça peut être totalement compatible avec la présentation du 10 Juin sur l'open education !

On en parle demain !

**Reply by AbcSxyZ** [reply]
Créer un ORCID ?

> Demander un DOI à la HES-SO

Tout dépend, vu que Zenodo en génère un automatiquement, tu n'es théoriquement pas obligé de passer par la HES-SO. doi.org, c'est des grosses bases de données {ID => url}, je pense que ce qui est le plus important est de pérenniser ce lien.

Je ne suis pas au clair sur la mise à jour du DOI, mais possible qu'en passant par votre école vous êtes plus susceptibles de pouvoir la faire, contrairement à Zenodo. Et les DOI de Zenodo contiennent Zenodo dedans (ex : 10.5281/zenodo.7911779), ce qui augmente la dépendance à la plateforme si demain vous êtes amenés à en utiliser une autre.

Si vous partagez le travail avec le DOI de Zenodo pendant X temps, que vous changez de plateforme de travail en cours de route et que vous ne pouvez pas mettre à jour les DOI, que se passe-t-il ? C'est au moins pour ce genre de situation que ça me paraît important de réfléchir à ce qui est fait, et peut-être que là ça pourrait devenir pertinant de passer par votre établissement. Je pense qu'il faut leur demander au passage comment/si il est possible de mettre à jour le DOI. Et quelle pertinence à une publication dans ArODES ? Une discussion avec un(e) bibliothécaire spécialisé(e) en open science de la HES-SO servirait à clarifier ces points.

> Concernant notre présentation, je travail sur un concept différent d'une présentation classique, je voulais pas trop de donner d'infos là dessus, pour garder la surprise mais ça peut être totalement compatible avec la présentation du 10 Juin sur l'open education !

Comme tu le sens pour la présentation et pour l'apporter le 10 juin. Dans tous les cas l'idée est qu'à côté des causeries sur l'école ("hacking the school") et de cet environnement pédagogique global sur les modèles ouverts, ça serait de permettre à chacun de proposer des ateliers (de découverte) sur un sujet avec un programme contributif pour donner la place à ce genre de présentation :)

**Reply by slrleao** [reply]
J'ai demandé à J. des contacts pour des bibliothécaire de la HES-SO qui s'y connaissent en open science et je vais poser ces questions. Je te tiens au courant. Merci !

**Comment by slrleao** [comment]
J'ai une question et réflexion sur le système de DOI et de publication d'open access en général.

DOI est donc une fondation qui gouverne le système de DOI. Ils utilisent les standards ISO, donc non ouvert. Ceci me parait en contradiction avec les principes dit "ouverts".

Pourquoi est-ce qu'on aurait besoin d'un DOI lorsqu'un URL marche à peu prêt de la même manière ? Ils garantissent que le lien dure pendant 50 ans, pourquoi ? Que se passe t'il si DOI ferme ou fait faillite ? Pourquoi est-ce que l'on doit passer par une institution comme la HES pour avoir un DOI ? Ce n'est pas ouvert et accessible par tou ?
 On doit donc faire confiance à une fondation "fermée" pour héberger un lien vers nos documents "ouverts" (open access/science). De ce que je comprends, le seul avantage est de facilité la recherche de notre document dans le milieu scientifique et aussi car c'est comme ça que les papiers scientifiques se font publier. Est-ce une bonne raison de faire la même chose ?

Je porte le même type de réflexions sur les plateformes d'open access de type Zenodo, etc. Pourquoi faire confiance à des tiers qui hébergent notre contenu ? Pareil, que se passe t'il si ils ferment ? On n'a pas la souveraineté sur nos publications. De ce que je comprends des modèles ouverts jusqu'ici c'est vraiment ce que l'on veut éviter.

Héberger nos documents sur notre site web et ensuite les distribuer ou publier avec des liens type URL de manière libres sur internet ne serait-il pas plus cohérent avec nos démarches ?

Beaucoup de questions, je ne sais pas si tu as des réponses !

**Reply by AbcSxyZ** [reply]
Ce sont de bonnes questions à avoir, et ça soulève plusieurs enjeux. Ce sont des systèmes antérieurs à (la démocratisation de) l'open science. Je crois avoir déjà entendu parler d'une alternative (mais pas sûr de mes souvenirs et je ne retrouve pas). En cherchant un peu, qui ne me semble pas être ce que j'avais vu, je suis tombé sur le [Decentralized identifier](https://www.w3.org/TR/did-core/) (DID) du W3C (organisme de standardisation du web).

Ces problèmes de dépendance à une entité amènent des gens à imaginer des alternatives, mais la question de la démocratisation de ces autres standards se pose alors.

La question centrale est la persistance de l'information, et l'URL a un risque plus important de changer, comme lors d'une refonte du site. Même si idéalement c'est bien de maintenir les url, c'est plus susceptible de bouger qu'un id relié à un document, sur laquelle on peut mettre à jour l'URL.

Le problème, c'est qu'une dépendance s'est installé pour des outils et des pratiques de la recherche, tu prends [prereview.org](https://prereview.org/) ça repose sur le DOI.

Est-ce que faciliter l'accès, le partage de l'information et son usage dans la sphère scientifique en se conformant à certaines pratiques actuelles reste désirable ?

L'enjeu est de savoir comment composer (ou non) avec certaines logiques pour permettre le développement d'autres. Personnellement, je ne suis pas dans une radicalité à utiliser que des systèmes purement ouverts et "décentralisés", cela me paraît aujourd'hui ajouter trop de difficultés sur un sujet déjà complexe à amener, la preuve on est sur GitHub. Un choix discutable, mais être aligné sur tout est un gros défi dans un monde contraire à ces logiques.

Existe-t-il un système "décentralisé" sans un minimum de centralisation (avec le numérique) pour former ces nœuds ? Est-ce que vous hébergez tout le contenu sur place à OKA avec vos serveurs ? Si oui, est-ce que vos serveurs sont produits par vos soins, avec une maîtrise complète de l'architecture ? Si votre contenu est hébergé sur place et que l'éco-campus brûle, que se passe-t-il ? Est-ce que vous pouvez vous faire confiance pour tout gérer et garantir cette pérennité ?

Reposer sur un seul endroit, quel qu'il soit, comportera des risques. Je t'invite à regarder la règle des 3-2-1 pour les sauvegardes, ça aidera peut-être à répondre à certaines questions. Est-ce que le site d'OKA me semble offrir une mise à disposition plus pérenne que Zenodo soutenue par le CERN ? Je ne te ferais pas l'affront de répondre 😉 Mais tout faire reposer sur Zenodo serait aussi une erreur.

Et il y a encore une fois la question de l'indexation de votre contenu par les moteurs de recherche (open access), si tout repose sur votre site, allez-vous implémenter le protocole OAI-PMH ou alors accepter qu'il soit uniquement trouvable par votre site ? Ou allez-vous héberger votre propre archive ouverte, ce qui est possible (mais une charge en plus) ?

Faut-il favoriser l'accessibilité et diffusion de la recherche en adoptant au moins une partie des pratiques de la communauté scientifique ou allez à l'encontre de tout de qui ne respecteraient par certains principes au risque de complexifier le développement de ces processus ? Est-ce qu'il faut aller affronter la complexité partout, comme en passant des heures à trouver et implémenter le système d'ids encore de niche, ou parfois reposer sur l'existant un peu dans l'idée où "c'est comme ça", ce qui n'exclut pas la critique ? Vous avez ce dilemme entre vos mains.

Pas de DOI et pas d'archive ouverte (indexée) avec OAI-PMH, c'est combiner 2 freins. 

Entre l'idéal et la réalité, je crois qu'il y a la frustration.

**Reply by slrleao** [reply]
Je te remercie pour tes réponses et tes explications. Je comprends bien tout l'enjeu et la difficulté de faire la transition actuellement.
C'est un peu les valeurs vs l'accessibilité et la diffusion de la recherche.

Bien sûr que si c'est la seule manière actuellement de partager l'information et d'avoir un maximum d'impact, c'est le chemin que l'on va prendre. Par contre ça ne nous empêche pas de rester critique sur ces incohérences et de regarder pour des alternatives (de type bittorent par exemple).

Je pense sincèrement que les technologies existantes aujourd'hui offrent énormément des possibilités pour la suite. Sans parler du fait que la société change de plus en plus vite sous l'effet de sa croissance. Des solutions telles que le réseau Bitcoin et Safe Network répondent à certaines de ces questions. Je t'en ai déjà parlé mais je t'invite, si ça t'intéresse, à regarder https://safenetwork.org/ qui est en développement.

Je pense qu-il faut utiliser ces "anciennes" méthodes (DOI, publications Zenodo, etc.) de publications en attendant de trouver des alternatives. Car à mon avis, si l'on continue à uniquement promouvoir les modèles ouverts à travers des moyens fermés, ça sera de plus en plus dur de sortir de cette dépendence. De la même manière que OKA cherche à repenser notre mode de vie au lieu d'innover sur des fondations instables.

Concernant la manière dont on gère notre contenu actuellement, tout se trouve sur un GIT repos, sur un clé USB. Bien-sûr c'est une solution temporaire et on va rapidement passer sur un GITLab avec un cloud "open source" et devoir penser à toutes ces questions d'hébergement.

Je suis au tout début de mon apprentissage sur ces dynamiques, de même que pour l'informatique en général ou je commence à apprendre petit à petit. Ce genre de discussions et de questions jouent un rôle important pour cette compréhension. Je te remercie pour ça !

**Reply by AbcSxyZ** [reply]
![coloriage-a-imprimer-equilibriste-au-cirque](https://github.com/Open-Models/Brique/assets/34010605/da040748-fd24-4fa3-be76-11d4c4c3c7f1)

**Reply by slrleao** [reply]
L'ambition est trop grande ? Hahahaha
On verra jusqu'où on peut aller ! C'est du game theory, on s'adapte à se qu'il se passe ;)
